main/webapp/modules/core/styles/common.less

```diff
@@ -436,3 +436,10 @@ p.body-text {
   height: 100%;
   overflow: hidden;
   }
+
+.unprintableCharacters{
```

Comment:
Also, for some reason, I think the margin-top is getting accumulated?  I preferred a setting of `margin-top: -3px;`

![image](https://user-images.githubusercontent.com/986438/77448209-7e654000-6dbe-11ea-92da-de26f0975e62.png)

Reply:
Double check all that in both FF and Chrome.